assets/src/components/activities/adaptive/components/authoring/LayoutEditor.tsx

```diff
@@ -511,22 +527,53 @@ const LayoutEditor: React.FC<LayoutEditorProps> = (props) => {
             onSubmit: defaultHandler,
             onResize: defaultHandler,
           };
+
           const disableDrag =
-            selectedPartId !== part.id ||
```

Comment:
I'm allowing an unselected part to be dragged/resized now. It seems like better UX to just drag an element instead of having to select it, and then start dragging.